idMap/idMapMount.go

```diff
@@ -177,9 +178,8 @@ func runIDMapMountCheckOnHost(dir string, checkOnOverlayfs bool) (bool, error) {
 
 	// Create a userns process that simply pauses until killed
 	execFunc := func() {
-		for {
-			c := make(chan struct{})
-			<-c
+		for i := 0; i < 3600; i++ {
```

Comment:
I suspect there's a better/more-elegant way to do this, but don't see anything wrong with this approach.